trunk/SUAVE/Components/Energy/Converters/__init__.py

```diff
@@ -18,4 +18,4 @@
 from Turbine import Turbine
 from Supersonic_Nozzle import Supersonic_Nozzle
 from Shaft_Power_Off_Take import Shaft_Power_Off_Take
-from Gearbox import Gearbox
```

Comment:
not used?

Reply:
Corrected, Issue Resolved.

Reply:
So Gearbox is not used?
No changes made

Reply:
it is used, I took it out by mistake